### ISSUE TITLE
Wrong core dependency?

### ISSUE THREAD

**Issue by nascentt** [chocolatey/ChocolateyGUI#888]
chocolateygui 0.18.1 has the dependency of `id="chocolatey-core.extension" version="1.3.3"`
however it seems to need "chocolatey-core.extension 1.3.5.1" otherwise the error "'Get-AppInstallLocation' is not recognized as the name of a cmdlet" occurs.

once updating from chocolatey-core.extension version=1.3.3 to 1.3.5.1 the issue no longer occurs installing and using chocolateygui
figured i'd raise this in case the requirement needs to be fixed in chocolateygui package

**Comment by gep13** [comment]
@nascentt I am not able to replicate a problem here.

Version 1.3.3 of the chocolatey-core.extension package also has the `Get-AppInstallLocation` cmdlet.

Can you provide a complete steps of steps to replicate the problem that you are seeing, including providing information about the other packages (with versions) that you have installed.  Thanks

**Comment by nascentt** [comment]
I'll have to reopen if it reoccurs. I've upgraded all hosts to 1.3.5.1 now so have no host displaying the error to test with.

the main host in particular that I saw exhibiting this issue had 1.3.3 and kept showing "Get-AppInstallLocation' is not recognized as the name of a cmdlet" until I upgraded it.
It's of course possible there was just a fault with the installed 1.3.3 and a reinstall of 1.3.3 would have also fixed it.
